pkg/schedule/schedulers/base_scheduler.go

```diff
@@ -146,3 +146,10 @@ func (s *BaseScheduler) IsDefault() bool {
 	}
 	return false
 }
+
+// IsFinished implements the Scheduler interface.
+func (*BaseScheduler) IsFinished() bool {
```

Comment:
If it's only used in test and log, please remove it.

Reply:
fixed